frontend/src/pages/UpdateUser.tsx

```diff
@@ -1,5 +1,139 @@
-import { Box } from '@mui/material';
+import {
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  Typography,
+  Button,
+  FormControl,
+} from '@mui/material';
+import Background from '../components/Background';
+import { useState } from 'react';
+import { Faculty } from '../types';
 
 export const UpdateUser = () => {
-  return <Box>This is the UpdateUser Page!</Box>;
+  const [userDetails, setUserDetails] = useState<{
+    emojis: string;
+    faculty: string;
```

Comment:
Let's set the type of the faculty from string to the Faculty enum. On line 21, we can just set our default value to a random Faculty (e.g. business)